### ISSUE TITLE
Treat account age limitation differently for buyers vs sellers

### ISSUE THREAD

**Issue by bitxat** [bisq-network/bisq#1476]
Hello guys,
as discussed on the keybase group earlier today, i will post my ideas about the lime limitation for new accounts.

@cbeams wrote:
> The account age witness feature we’re discussing here is designed specifically to deter bank account thieves from using Bisq. If you have just gotten access to a stolen account, it will not be economical for you to (a) do dozens of trades trying to clear it out or (b) wait two months for your account to mature. You will likely go elsewhere to dump those funds.
> [...]
> We could possibly treat account age differently for buyers vs sellers. 

So my point is that the BUYER of BTC should be limited but not the SELLER. The seller has BTC and wants the receive EUR on his bank account, there no limit should be applied since he is not sending money from a stolen account, he just wants to sell his BTC without limitation. The limitation should only apply to the buyer of the btc since he could eventually sending the funds from a stolen account, not the buyer. Its is logical to me.

I hope you can implement that the account age is treated differently for buyers vs sellers in the next release.

Thank you for your hard work. BISQ = Freedom of trade with privacy for the buyer and seller.

**Comment by cbeams** [comment]
Thanks, @bitxat. Note that I updated your description above to quote what I wrote in the Keybase chat vs. what you yourself were writing.

@ManfredKarrer, did you consider asymmetric treatment of account age for buying vs. selling when originally designing the feature? Did you rule this out for some reason? I haven't spent any time thinking about this, but @bitxat's request isn't unreasonable at a glance.

**Comment by ManfredKarrer** [comment]
Yes I was considering it but kept it out because of more complexity mainly for the UX and offerbook.

**Comment by Chronos23432** [comment]
I am missing deals to purchase Btc at a low price because of account age limit and the fact that sellers post trades that require me to wait at least another month. So far, I traded successfully with many sellers, but this limitation of account age is bothering. I traded in two countries with one of them having extremely strict banking rules where I have to call each time(!), answer identifying questions, to ensure payment to seller is approved.

Proposal: If a user registered several accounts to trade with AND has traded with at least two of them, then the information (first and last name provided when creating account) could be used to assure that such user isn't a thief but a credible buyer.

How does that sound?
